.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - stable/*
+      - release-*
+      - trying
+      - staging
+  pull_request: {}
+  workflow_dispatch: {}
+
+jobs:
+  test:
+    name: Test
+    uses: ./.github/workflows/test.yml
+    secrets: inherit
+  code-quality:
+    name: Code Quality
+    uses: ./.github/workflows/code-quality.yml
+    secrets: inherit
+  deploy:
+    name: Deploy
+    if: github.ref == 'refs/heads/main'
+    needs:
+      - code-quality
+      - test
+    uses: ./.github/workflows/deploy.yml
+    secrets: inherit
+  bors-check:
```

Comment:
:question: Couldn't bors just check for `Continuous Integration`? Or can it only check jobs and not workflows? I'm not super familiar what bors is looking at.

Reply:
No, it can only check job, otherwise it'd be easy to just add a check for the test and code-quality workflows.